style.css

```diff
@@ -0,0 +1,77 @@
+* {
```

Comment:
- [ ] The design should look similar to the Figma one, kindly consider adding a height of 100vh to the headline section so it looks like the Figma design

|Your design|Figma design|
|-|-|
|![Screenshot_20220630_044728](https://user-images.githubusercontent.com/25227022/176590231-a3aac0a7-49a3-4cec-a769-3e6289ff00e8.png)|![Screenshot_20220629_210335](https://user-images.githubusercontent.com/25227022/176590232-e81d1f55-241c-4953-9400-98c402e79785.png)|